src/interface/report/Results/PlayerInfoConduits.tsx

```diff
@@ -0,0 +1,30 @@
+import { ConduitLink } from 'interface/index';
+import { Conduit } from 'parser/core/Events';
+
+interface PlayerInfoConduitsProps {
+  conduits: Conduit[] | undefined;
+}
+
+const PlayerInfoConduits = ({ conduits }: PlayerInfoConduitsProps) => (
+  <div className="player-details-talents">
+    <h3>Conduits </h3>
+    {conduits?.map((conduit) => (
+      <div
+        key={conduit.spellID}
+        className="col-md-12 flex-main"
+        style={{ textAlign: 'left', margin: '5px auto' }}
+      >
+        <div className="row">
+          <div className="col-md-10">
+            <ConduitLink
+              id={conduit.spellID}
+              iconStyle={{ width: '2em', height: '2em', border: '1px solid', marginRight: 10 }}
```

Comment:
I would suggest moving the iconStyle styling to a seperate (s)css file to avoid this all of the soulbinds having a style attribute with the above styling.
![image](https://user-images.githubusercontent.com/3527340/184625852-85d27988-515f-47f4-ac5c-e634e8446047.png)